Updated the PEM Upgrade guide (#2151)
EnterpriseDB/docs: product_docs/docs/pem/8/pem_upgrade/01_upgrading_pem_installation/05_configuring_the_pem_server_on_linux_platforms.mdx

/usr/edb/pem/bin/configure-pem-server.sh
```

> While running the configure script, it will take the values from the old PEM Server configuration file and run the script.

Reviewer: ```suggestion
The configure script uses the values from the old PEM Server configuration file while running the script.
```

After executing the PEM server configuration file, use your version-specific service control command to restart the `httpd` service.

> For detailed information about using an RPM package to install or configure the PEM server or PEM Agent, please see the [PEM Linux Installation Guide](https://www.enterprisedb.com/docs/pem/latest/pem_inst_guide_linux/).

Reviewer: If you change the names of the install guide, update this (and any other) references to it.

Author: I have changed only the the titles inside the index.mdx file so the link references do not need to be updated
